rai_test_utils/rai_test_utils/utilities/utils.py

```diff
@@ -2,8 +2,11 @@
 # Licensed under the MIT License.
 
 import os
+import shutil
 import uuid
 
+DOWNLOADED_DATASET_DIR = 'datasets.4.27.2021'
```

Comment:
That seems oddly specific. Isn't it going to change?

Reply:
we use a specific zip folder with the date it was created for these datasets (that are put on our blob storage for tests)

Reply:
Right, so if the dataset changes this need to be updated. How will we remember?